.evergreen/config/test/ocsp-7.0.yml

```diff
@@ -0,0 +1,389 @@
+tasks:
+  - name: "ocsp-test_1-rsa-delegate-7.0"
```

Comment:
The OCSP tasks may be candidates for using a task group. This would allow us to only fetch the build once, install the correct MongoDB version, then run all tasks sequentially on the same build host. This would further cut down resource usage. However, we'll want to make some improvements to mongodb-orchestration to make sure we can set up and tear down OCSP responders properly. We might also have to split up the mongo-orchestration files to allow for separate functions for downloading MongoDB, starting/stopping orchestration, and starting/stopping a deployment.